### DISCUSSION TITLE
Grid

### DISCUSSION THREAD

**Discussion by boxadesign** [ONSdigital/design-system#72] [Feedback]
Use this place to discuss the [grid](https://ons-design-system.netlify.com/styles/grid) in the [ONS Design System](https://ons-design-system.netlify.com)

**Comment by jrbarnes9** [comment]
Lost the grid helpers - need to re-instate but in the pat-lib

**Reply by jrbarnes9** [reply]
#468 resolved

**Comment by jrbarnes9** [comment]
Loose grid needs greater negative margin

**Reply by jrbarnes9** [reply]
#467 resolved

**Comment by jrbarnes9** [comment]
Grid examples lost background color
![Screenshot 2020-08-17 at 16 35 30](https://user-images.githubusercontent.com/43346934/90414546-d53e1f80-e0a7-11ea-8a38-3d4d3762e4ab.png)

**Reply by jrbarnes9** [reply]
#924 resolved

**Comment by jrbarnes9** [comment]
Grid: Document new additional breakpoint (400px) 
`xxs` breakpoint missing from DS grid documentation

**Reply by jrbarnes9** [reply]
#1367 resolved

**Comment by mr-nick17** [comment]
Would it be possible to add flex utility classes that you want to target with media queries? i.e. You want to apply flex grid to something only in smaller viewports. The ONS website, [dp-design-system](https://github.com/ONSdigital/dp-design-system/blob/8a0b3d6639b584adaccb036f4772db03822fe966/src/scss/dp/overrides/design-system/_foundations.scss#L8), has something similar

**Comment by henryjameslau** [comment]
Typo in the instructions `ons-grid__col--flex` should be `ons-grid--column`.

**Reply by Olatunde-Alalade-ONS** [reply]
Thanks Henry. Could you kindly provide a URL to the page for reference

**Reply by henryjameslau** [reply]
It's here in the flexbox section https://service-manual.ons.gov.uk/design-system/foundations/grid#using-the-grid-system

![image](https://github.com/user-attachments/assets/5e880f7c-bae6-4827-8f68-7e3e248128d6)

**Reply by Olatunde-Alalade-ONS** [reply]
Thanks.. Noted for review and update.
